extending-valtimo/document/configuring-search-fields.md

```diff
@@ -72,34 +76,34 @@ An existing search field can be deleted by calling the following endpoint:
 value of the key that should be deleted.
 
 ## Allowed values
-The datatype, fieldtype and matchtype only allow specific values. This paragraph will mention all available values
+The dataType, fieldType and matchType only allow specific values. This paragraph will mention all available values
 and what the search types do.
 
-### Datatype
-The allowed values for datatype are:
+### DataType
```

Comment:
This one's a bit iffy. I understand capitalizing the T, but we do this in code because we consider the two to be different words. Since this is the title of a section, maybe use "Data type" instead?